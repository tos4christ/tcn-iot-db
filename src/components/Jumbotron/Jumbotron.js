import React from "react";
import { Link } from "react-router-dom";
import handMockUp from "../../assets/img/hand-mockup.png";

const Jumbotron = () => {
  return (
    <section className="hero-area" id="home">
      <div className="container">
        <div className="row">
          <div className="col-lg-7">
            <div className="hero-area-content">
              <h1>TCN Dispute Resolution App</h1>
              <p>
                A tool for Discos to submit resolutions
                needed to be addressed by TCN on their equipment
                interfacing with us
                 <br />
                <br />

                
                Available on andriod and IOS devices.
              </p>

              <Link to="/" className="appao-btn">
                <i className="icofont icofont-brand-android-robot"> </i>
                Google Play
              </Link>
              <Link to="/" className="appao-btn">
                <i className="icofont icofont-brand-apple"> </i>
                App Store
              </Link>
            </div>
          </div>
          <div className="col-lg-5">
            <div className="hand-mockup text-lg-left text-center">
              <img src={handMockUp} alt="Hand Mockup" />
            </div>
          </div>
        </div>
      </div>
    </section>
  );
};

export default Jumbotron;
