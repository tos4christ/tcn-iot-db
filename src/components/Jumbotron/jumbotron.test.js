import React from "react";
import renderer from "react-test-renderer";
import "@testing-library/jest-dom/extend-expect";
import Jumbotron from "./Jumbotron";
import { Router } from "react-router-dom";

it("renders jumbotron", () => {
  const historyMock = {
    push: jest.fn(),
    location: {},
    listen: jest.fn(),
    createHref: jest.fn(),
  };
  let tree = renderer
    .create(
      <Router history={historyMock}>
        <Jumbotron />
      </Router>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
