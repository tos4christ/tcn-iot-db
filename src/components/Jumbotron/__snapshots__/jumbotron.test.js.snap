// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders jumbotron 1`] = `
<section
  className="hero-area"
  id="home"
>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="col-lg-7"
      >
        <div
          className="hero-area-content"
        >
          <h1>
            Road Safety is our utmost priority
          </h1>
          <p>
            RoadMaster helps safety and security personnels respond swiftly to unplanned events; accidents, crashes, explosions, etc. It also suggests alternative routes to avoid unnessecary traffic caused during these disasters. RoadMaster is safe and easy to use. 
            <br />
            <br />
            Available on andriod and IOS devices.
          </p>
          <a
            className="appao-btn"
            onClick={[Function]}
          >
            <i
              className="icofont icofont-brand-android-robot"
            >
               
            </i>
            Google Play
          </a>
          <a
            className="appao-btn"
            onClick={[Function]}
          >
            <i
              className="icofont icofont-brand-apple"
            >
               
            </i>
            App Store
          </a>
        </div>
      </div>
      <div
        className="col-lg-5"
      >
        <div
          className="hand-mockup text-lg-left text-center"
        >
          <img
            alt="Hand Mockup"
            src="hand-mockup.png"
          />
        </div>
      </div>
    </div>
  </div>
</section>
`;
